docs/installation.md

```diff
@@ -0,0 +1,136 @@
+# Installation
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+## Prerequisites
+
+The project has the following [dependencies](#dependencies):
+
+* [Algolia](#algolia)\*
+* [Auth0](#Auth0)
+* [Google Cloud](#google-cloud)\*
+* [Graphcool](#graphcool)\+
+* [Mailgun](#mailgun)\*
+
+> \* _optional, but recommended_
+>
+> \+ _local deployment available_
+
+## Installing
+
+First, clone the repository
+
+```bash
+# Path: ./
+git clone https://github.com/Zenika/FAQ
+```
+
+Then, install the dependencies of the project
+
+```bash
+# Path: ./FAQ/
+npm install
+```
+
+## Configuration
+
+Before running the project, you will need to configure your environment variables from the [dependencies](#dependencies)
+
+```bash
+# Path: ./FAQ/
+cp .env.local.example .env.local
+vim .env.local
+
+cp server/.env.local.example server/.env.local
+vim server/.env.local
+```
```

Comment:
As an aspiring contributor I would be lost here. Either I'm not familiar with `.env` files and I don't know what to do with them, or I don't know what values to write in it. A simple "Keep reading to know what values to write in this file." would cover the second case.